modules/siddhi-core/src/test/java/io/siddhi/core/query/table/DeleteFromTableTestCase.java

```diff
@@ -43,6 +43,48 @@ public void init() {
         eventArrived = false;
     }
 
+    @Test
+    public void deleteFromTableTest0() throws InterruptedException {
+        log.info("deleteFromTableTest1");
```

Comment:
name the test consistency else it will be hard to debug this should be `deleteFromTableTest0`